### ISSUE TITLE
Feedback on "Styling React"

### ISSUE THREAD

**Issue by Foxandxss** [survivejs/webpack_react#142]
Lets start saying that the chapter idea is nice, but in my opinion it is not good as a chapter 8.

The chapter is mostly two different things:

1) You finally add the styling to the app.
2) You explain dozen of different alternatives to how to style a react app.

Personally I would move the first point to an early stage of the book, I would probably mix it in other chapters.

The webpack config is in place earlier where we put that yellow background. I would also add the font in there.

The explanation of the `extract-text-webpack-plugin` is suitable to be added at the beginning as well when we set webpack for the first time.

As for the lane and notes css, I would marge them somewhere. Some books likes to add all the styling at first and then forget (the book is not about CSS after all). You could also add the CSS after creating the lanes.

When we create an app, we normally leave the CSS for the end, but I think that for a book, the enduser needs to see it more "pretty" as early as possible, as a way of "keep the user reading".

This would cover point 1.

For the rest of the chapter, the dozen approaches, I would create an appendix and move everything in there.

At the end of the day, the book is a serie of steps to start from an empty folder to a fully functional app. If any explanation is outside the "scope" of that, you should make an appendix for that. Maybe not for a "side note" but this is way beyond a side note.

So it is more like a "If you want to know more ways of managing styles on react, appendix XXX"

TL;DR Merge the kanban stuff into other chapters and the "not kanban stuff" into an appendix.

What do you think?

**Comment by bebraw** [comment]
Good points! I agree it's a good idea to push theory bits later. That will make the book feel more integrated and focused. It's about the project after all.

**Comment by bebraw** [comment]
I integrated the styling bits to the end of "From Notes to Kanban". Now styling is discussed in detail at an appendix. I feel this improved the flow of the book a lot. Thanks!
